extension/scripts/content/travelagency/ttTravelAgency.js

```diff
@@ -1,24 +1,41 @@
 "use strict";
 
-(() => {
-	for (const type of document.findAll(".travel-agency .torn-tabs > li:not([data-state='disabled'])")) {
-		type.addEventListener("click", async () => {
-			triggerCustomListener(EVENT_CHANNELS.TRAVEL_SELECT_TYPE, { type: type.find(".travel-name").innerText.toLowerCase() });
+(async () => {
+	if (await checkMobile() || await checkTablet()) {
+		document.find(".travel-agency").addEventListener("click", () => {
+			if (
+				event.target.closest(".travel-info-table-list[aria-selected*='true'][role*='tab']")
+			) {
+				triggerCustomListener(EVENT_CHANNELS.TRAVEL_SELECT_COUNTRY, null);
+			}
 		});
-	}
-	for (const flag of document.findAll(".travel-agency .raceway")) {
-		flag.addEventListener("click", async () => {
-			const container = document.find(".travel-agency > div[aria-expanded='true'] .travel-container.full-map[style='display: block;']");
-			if (container) {
-				await new Promise((resolve) => {
-					new MutationObserver((mutations, observer) => {
-						observer.disconnect();
-						resolve();
-					}).observe(container, { childList: true });
-				});
-			} else await requireElement(".travel-agency > div[aria-expanded='true'] .travel-container.full-map[style='display: block;']");
-
-			triggerCustomListener(EVENT_CHANNELS.TRAVEL_SELECT_COUNTRY, { country: flag.dataset.race });
+		document.find(".torn-tabs").addEventListener("click", () => {
+			if (
+				event.target.closest(".torn-tabs li[aria-selected='true']")
+			) {
+				triggerCustomListener(EVENT_CHANNELS.TRAVEL_SELECT_TYPE, null);

```

Comment:
If there are no details to pass through, simply skip this parameter and just pass 1 instead of the 2nd one being `null`.

Reply:
And you should also include the tab if possible, as it's passed in the other trigger so not providing it could cause bugs.